crates/storage/store_db/rocksdb.rs

```diff
@@ -1135,12 +1151,26 @@ impl StoreEngine for Store {
         .map_err(|e| StoreError::Custom(format!("Task panicked: {}", e)))?
     }
 
-    fn get_account_code(&self, code_hash: H256) -> Result<Option<Bytes>, StoreError> {
+    fn get_account_code(&self, code_hash: H256) -> Result<Option<Code>, StoreError> {
         let hash_key = code_hash.as_bytes().to_vec();
-        self.read_sync(CF_ACCOUNT_CODES, hash_key)?
-            .map(|bytes| AccountCodeRLP::from_bytes(bytes).to())
-            .transpose()
-            .map_err(StoreError::from)
+        let Some(bytes) = self.read_sync(CF_ACCOUNT_CODES, hash_key)? else {
```

Comment:
We could use the `get_pinned_cf` rocksdb method here to avoid copies before deserializing.